Clicker/Views/PollBuilder/AskQuestionCell.swift

```diff
@@ -12,55 +12,84 @@ import UIKit
 protocol AskQuestionCellDelegate: class {
     func askQuestionCellDidUpdateEditable(_ editable: Bool)
     func askQuestionCellDidUpdateText(_ text: String?)
+    func askQuestionCellDidUpdateHeight(_ height: CGFloat)
 }
 
 class AskQuestionCell: UICollectionViewCell {
 
     // MARK: - View vars
-    var textField: UITextField!
+    var questionTextView: UITextView!
+    var charCountLabel: UILabel!
 
     // MARK: - Data vars
     weak var delegate: AskQuestionCellDelegate?
-
+    
```

Comment:
do you have your trim trailing whitespace enabled?
![](https://pbs.twimg.com/media/DTawMEMWsAAbbbs.png)

Reply:
Enabled! - Thanks for the tip!